lms/djangoapps/courseware/module_render.py

```diff
@@ -335,6 +337,38 @@ def get_module(user, request, usage_key, field_data_cache,
         return None
 
 
+def display_access_messages(user, block, view, frag, context):
```

Comment:
there are some unused arguments here, not sure if we need to pass them into the function? if we do you'll just need to fix the pylint    

this function could also use a docstring and some comments, it is somewhat hard to follow

Reply:
Pylint ignored because they are required for an xblock wrapper. Added doc string.